cli.js

```diff
@@ -60,8 +55,8 @@ const scriptArgs = process.argv.slice(3)
   .join(' ')
 
 searchForScript(__dirname, scriptPath)
-  .then(
-    foundPath => executeScript(foundPath, scriptArgs),
-    triedPaths => throwSearchFail(scriptPath, triedPaths)
-  )
-  .catch(() => process.exit(1))
+  .then(foundPath => executeScript(foundPath, scriptArgs))
+  .catch(e => {
+    console.error(e.toString())
+    process.exit(1)
+  })
```

Comment:
Just for education, `.catch` is not just sugar for `.then(null, fn)`. Catch will catch errors thrown during preceding `.then` success handlers, i.e.:

```js
somePromise
  .then(_ => throw new Error('hey'))
  .catch((err) => // catches ^^)
```

vs.

```js
somePromise
  .then(
    () => throw new Error('hey!'),
    () => // no error caught!
  )
```